source/_integrations/pid_thermostat.markdown

```diff
@@ -0,0 +1,145 @@
+- For kd, keep this number to 0 until kp and ki are set. Now you can use the kd to prevent the regulator from overshoot. Only increase in small steps.
+
+## Configuration
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+Name:
+  description: The name that the PID thermostat should have. You can change it again later.
+Heater device entity:
+  description: Heater- or cooler number entity. Typically, the [slow_pwm number entity][slow_pwm] can be used to create a number controlling a switch.
+Heater device entity:
+  description: Temperature sensor, used for input signal.
+Proportional gain factor (Kp):
+  description: Proportional gain factor, directly gaining the error to compensate the fault (Kp).
+Integration factor (Ki):
+  description: Integration factor, reducing offset fault over time (Ki).
+Differential factor (Kd):
+  description: Differential factor, damping the overshoot (Kd).
+Thermostat mode: 
+  description: Select if the thermostat should be a cooler or a heater.
+Duration between controller cycles
+  description: Cycle time that elapses between two controller cycles.
+{% endconfiguration_basic %}
+
+## YAML Configuration
+
+Alternatively, this integration can be configured and set up manually via YAML.
+To enable the PID thermostat entity in your installation, add the
+following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: pid_thermostat
+    name: Kitchen thermostat
+    heater: number.floor_heater
+    sensor: sensor.kitchen_temperature
+``` 
+
+{% configuration %}
+name:
+  description: Name of the PID thermostat.
+  required: true
+  type: string
+heater:
+  description: `entity_id` for heater or cooler. Must be a number device. Typically, the [slow_pwm number entity][slow_pwm] can be used to create a number controlling a switch.
+  required: true
+  type: string
+sensor:
+  description: `entity_id` for temperature sensor.
+  required: true
+  type: string
+kp:
+  description: Proportional gain factor, directly gaining the error to compensate the fault (Kp).
+  required: false
+  default: 100.0
+  type: float
+ki:
+  description: Integration factor, reducing the offset fault over time (Ki).
+  required: false
+  default: 0.1
+  type: float
+kd:
+  description: Differential factor, damping the overshoot (Kd).
+  required: false
+  default: 0.0
+  type: float
+ac_mode:
+  description: Thermostat mode. 
+  required: false
+  default: direct
+  type: string ('cool' or 'heat')
+min_temp:
+  description: Minimal temperature setpoint in °C.
+  required: false
+  default: 7
+  type: float
+max_temp:
+  description: Maximal temperature setpoint in °C.
+  required: false
+  default: 35
+  type: float
+cycle_time:
+  description: Cycle time for the internal PID controller.
+  required: false
+  default: {"seconds": 30}
```

Comment:
```suggestion
  default: "{'seconds': 30}"
```
Not 100% sure about this. Could you check and see if this fixes the pipeline run?